### DISCUSSION TITLE
Reservation no longer available

### DISCUSSION THREAD

**Discussion by AZenobi** [Alkaar/resy-booking-bot#117] [General]
First off, love the project!!

Tried to test this on a reservation I can see as available, however when running the bot, the resy client returns the "Reservation no longer available" message.
Has anyone encountered this or does anyone have any ideas how to troubleshoot?

Attached conf - auth token and api key.
[resyConfig.txt](https://github.com/Alkaar/resy-booking-bot/files/11437215/resyConfig.txt)

**Comment by Alkaar** [comment]
When exactly does it say `Reservation no longer available`?

Also the 00 for `snipeTime.minutes` is kind of weird. You only need one zero. Not sure if that matters honestly.

**Reply by AZenobi** [reply]
Thanks for your response! Yes that's the message I'm getting. Here's the full log. I can see the available slot on the resy page however I'm still getting the "Reservation no longer available" message.

[INFO ] 2023-05-13 11:48:07.839-04:00 ResyBookingWorkflow:18 - Taking the shot...
[INFO ] 2023-05-13 11:48:07.840-04:00 ResyBookingWorkflow:19 - (â•¥Ã©Î“Ã‡â•› Î“Ã»â•¡Î“Ã‡â•›)âˆ©â••â•—Ï€Ã¢Ã§Î“Ã²Ã‰Î£â••Ã‡ (â•¦ÃœÎ“Ã»â•œâ•¦ÃœÎ“Ã‡Ã–!)/
[INFO ] 2023-05-13 11:48:07.840-04:00 ResyBookingWorkflow:20 - Attempting to snipe reservation
[INFO ] 2023-05-13 11:48:08.001-04:00 ResyClient:197 - Config Id: rgs://resy/4956/644048/2/2023-05-16/2023-05-16/19:00:00/2/Bar Seat
[INFO ] 2023-05-13 11:48:08.067-04:00 ResyClient:79 - Payment Method Id: ***
[INFO ] 2023-05-13 11:48:08.069-04:00 ResyClient:88 - Book Token: ***
[INFO ] 2023-05-13 11:48:08.087-04:00 ResyClient:136 - Missed the shot!
[INFO ] 2023-05-13 11:48:08.087-04:00 ResyClient:137 - Î“Ã¶â•—Î“Ã¶Ã¼Î“Ã¶â•— âˆ©â••â•¡ \(â”¬â–‘Î“Ã»Ã­â”¬â–‘)/ âˆ©â••â•¡ Î“Ã¶â•—Î“Ã¶Ã¼Î“Ã¶â•—
[INFO ] 2023-05-13 11:48:08.087-04:00 ResyClient:138 - Reservation no longer available
[INFO ] 2023-05-13 11:48:08.087-04:00 ResyBookingBot$:53 - Shutting down Resy Booking Bot

**Reply by sagraziano** [reply]
> When exactly does it say `Reservation no longer available`?
> 
> Also the 00 for `snipeTime.minutes` is kind of weird. You only need one zero. Not sure if that matters honestly.

I am having the same issue as the OP 

`[INFO ] 2023-05-18 10:58:07.979-04:00 ResyBookingWorkflow:18 - Taking the shot...
[INFO ] 2023-05-18 10:58:07.979-04:00 ResyBookingWorkflow:19 - (҂‾ ▵‾)︻デ═一 (˚▽˚’!)/
[INFO ] 2023-05-18 10:58:07.979-04:00 ResyBookingWorkflow:20 - Attempting to snipe reservation
[INFO ] 2023-05-18 10:58:08.064-04:00 ResyClient:197 - Config Id: rgs://resy/70910/2130489/5/2023-05-28/2023-05-28/11:00:00/2/Indoor
[INFO ] 2023-05-18 10:58:08.098-04:00 ResyClient:79 - Payment Method Id: ***
[INFO ] 2023-05-18 10:58:08.098-04:00 ResyClient:88 - Book Token: ***
[INFO ] 2023-05-18 10:58:08.113-04:00 ResyClient:136 - Missed the shot!
[INFO ] 2023-05-18 10:58:08.114-04:00 ResyClient:137 - ┻━┻ ︵ \(°□°)/ ︵ ┻━┻
[INFO ] 2023-05-18 10:58:08.114-04:00 ResyClient:138 - Reservation no longer available
[INFO ] 2023-05-18 10:58:08.114-04:00 ResyBookingBot$:53 - Shutting down Resy Booking Bot`

**Reply by derekhctam** [reply]
I'm having the same issue here and running everything correctly, is there a way people have been able to resolve this?

**Reply by GraysonHorn** [reply]
Same issue for me.  Getting an "Internal Server Error" response when trying to bookReservation.  Testing it on a known no credit card required, all reservations open restaurant.

Update:
Added the User-Agent to header and that fixed it.  See #137

**Reply by sagraziano** [reply]
I ended up coding my own bot, willing to share it if you like. My discord is bigsteve3

**Reply by IsaacKatong** [reply]
If you have a bot which works that would be great. I am worried resy put in a blocker to prevent bots.

**Reply by AZenobi** [reply]
@Alkaar i get "Reservation no longer available"  after the loga at the end of when the bot attempts to take a shot (see below). Not this was working for a while (after adding the user agent but) then stopped working again around early July per my observation.

- The book token is ultimately a different one than specified in the config, could this be causing an issue?
- Has anyone here been able to successfully book something in the last 3 weeks?

[IJ][INFO ] 2023-07-28 13:58:58.048-04:00 ResyBookingWorkflow:18 - Taking the shot...
[INFO ] 2023-07-28 13:58:58.049-04:00 ResyBookingWorkflow:19 - (â•¥Ã©Î“Ã‡â•› Î“Ã»â•¡Î“Ã‡â•›)âˆ©â••â•—Ï€Ã¢Ã§Î“Ã²Ã‰Î£â••Ã‡ (â•¦ÃœÎ“Ã»â•œâ•¦ÃœÎ“Ã‡Ã–!)/
[INFO ] 2023-07-28 13:58:58.049-04:00 ResyBookingWorkflow:20 - Attempting to snipe reservation
[INFO ] 2023-07-28 13:58:59.476-04:00 ResyClient:197 - Config Id: rgs://resy/4956/644049/2/2023-08-18/2023-08-18/20:00:00/2/Bar Seat
[INFO ] 2023-07-28 13:58:59.552-04:00 ResyClient:79 - Payment Method Id: 16605206
[INFO ] 2023-07-28 13:58:59.554-04:00 ResyClient:88 - Book Token: JYBIBbxzpn4iBqLNBtb7tUs|JGOzi2Jkx4kQiQpQGP_qEWd7WY_VajBqWqZf1_7MLBVsR6s2hnuNo2Zy98O9rdEvmkL7mDtbcpP4N5ffCIe6Ed5YgcVLSL4Ua2lGhqRF9k4UTM0MXiwE_1__ohfroUZs3x0Q1S9W2wTUHpI9pjlwdI8Bzst_Cpfq8BAArWfZ3Wt0voXNUo6uWZ3T7tA3JY0y1BmO0wcLuDS7RpJFpsUgxq|S273|DuFCVkHEdJBz06kmotS|IvmckJGtGKMEMQTiEOz9UQOFVs5aVAYr036OziRCDD9Zwq2Th0TM6a6b|9En76XATdqTDeLMXv0SPVLh8KVbqrlPUvdDp1wSsJ_seUXXT2wmjGAu3J_xqoxIeICcSb4zNhSPgK5kD6MhTyrBvpdYYtQdsjigeWZu45u4wQs2jApyZQ|ReiP7soGSszngyot4Uzbtc33v2IWGXrvGMVCPAHkQfxR|pfSlBu51nC4gVKNMzG8cQSofRN5F_tFYfEmKGf0waSl1zxDjMMk9745VIPLvHLms|I4JEG4zXJQHQ678uYIZF1eNGYFWCOg0QDju7abWFYAFMvc0XdO1Q3zXOMP0T4329Bb6tuZW96j1R_xLW3uVod3sRKzhSu_S6Mds4FVd8nh_SjpkkJAUW5_hjMDpVgTOVu1FVtREbNZfhzaMkifQSMpL7NF2-ecf8cfa3b205e162f4bd043a1e3f15929ce5a96e0e77dd61ad3042a8
[INFO ] 2023-07-28 13:58:59.845-04:00 ResyClient:136 - Missed the shot!
[INFO ] 2023-07-28 13:58:59.845-04:00 ResyClient:137 - Î“Ã¶â•—Î“Ã¶Ã¼Î“Ã¶â•— âˆ©â••â•¡ \(â”¬â–‘Î“Ã»Ã­â”¬â–‘)/ âˆ©â••â•¡ Î“Ã¶â•—Î“Ã¶Ã¼Î“Ã¶â•—
[INFO ] 2023-07-28 13:58:59.846-04:00 ResyClient:138 - Reservation no longer available
